modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -9,6 +9,12 @@ import Foundation
 import MobileCoreServices
 #endif
 
+public protocol URLSessionProtocol {
```

Comment:
@leszek-s instead of public, could you please use `{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}}`?

Reply:
I've changed it as suggested.